### PR TITLE
add source entry for simple_grasping

### PR DESCRIPTION
**Pull request by mikeferguson**
This package was released in ROS1, getting ready for ROS2 release - first step is getting name approval before creating ros2-gbp config

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Review by audrow — APPROVED** [review]
I'm okay with keeping it the same name as in ROS 1. I'd like to wait for a second opinion before merging.

**Review by clalancette — APPROVED** [review]

**Pull request merged**
